<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 244, 255);
border-radius:20px;
border:2px solid;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="txtNumeroDocumento">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>100</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>110</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);
font: 18pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Nº</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Código Producto:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Fecha:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Nombre y Apellidos:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Código Cliente:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCodigoCliente">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtFecha">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>150</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCodigoProducto">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>190</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Descripción:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDescripcion">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNombres">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(12, 12, 12);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Stock:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGenerar">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>580</y>
         <width>151</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}

color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Generar Venta</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tblDetalle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>951</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(12, 12, 12);
</string>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>156</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nro</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CÓDIGO PRODUCTO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>DESCRIPCIÓN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PRECIO</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>CANTIDAD</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>TOTAL</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="btnImprimirFactura">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>580</y>
         <width>151</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}</string>
       </property>
       <property name="text">
        <string>Imprimir Factura</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCerrar">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>580</y>
         <width>151</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

	background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}</string>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>10</y>
         <width>161</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../img/compras.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>481</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
font: 40pt &quot;Arial&quot;;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Punto de Venta &quot;GICSON&quot;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
font: 20pt &quot;Arial&quot;;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Venta de Productos Tecnológicos</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuscarCliente">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}

color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/icono_buscar.png</normaloff>../img/icono_buscar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBuscarProducto">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}

color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../img/icono_buscar.png</normaloff>../img/icono_buscar.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Precio:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCantidad">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>210</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtStock">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>210</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPrecio">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>210</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAgregar">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>250</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(58, 184, 255);
font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(0, 255, 69);
font:14pt &quot;Monaco&quot;;

}

color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>540</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>SubTotal:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>570</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>IGV (18%)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>600</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgb(25, 25, 25);</string>
       </property>
       <property name="text">
        <string>Total a Pagar:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtSubTotal">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>540</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtIgv">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>570</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTotalPagar">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>600</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(25, 25, 25);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
